pkg/util/arm/resources.go

```diff
@@ -43,24 +42,29 @@ func (r ArmResource) String() string {
 // ParseArmResourceId take the resourceID of a child resource to an OpenShiftCluster
```

Comment:
```suggestion
// ParseArmResourceId takes in an ARM resource ID and returns an ArmResource object representing that resource. It supports up to two levels of subresource nesting.
```

Not related to your work in this PR, but I noticed this comment wasn't a full sentence, so I'm proposing a better doc comment.